src/main/java/org/fundacionjala/coding/daniel/Kata7.java

```diff
@@ -0,0 +1,22 @@
+package org.fundacionjala.coding.daniel;
+
+/**
+ * A class which contains 'fizzBuzz' method.
+ */
+public class Kata7 {
+
+    private static final int FIZZ = 3;
+    private static final int BUZZ = 5;
+
+    /**
+     * @param number a integer.
+     * @return a String whether the number is 'Fizz', 'Buzz' or 'FizzBuzz'.
+     */
+    public String fizzBuzz(int number) {
+        final String value = String.valueOf(number);
+        return number % FIZZ == 0 || value.contains("3")
+            ? number % BUZZ == 0
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)